src/sentry/utils/outcomes.py

```diff
@@ -139,8 +104,6 @@ def track_outcome(org_id, project_id, key_id, outcome, reason=None, timestamp=No
                 }
             ),
         )
-        # outcomes originating in Sentry (sent by this function)
-        mark_outcome_signal_sent(project_id=project_id, event_id=event_id)
```

Comment:
We need to mark somewhere the outcomes coming from Sentry so that we don't double send the signal ( if we remove mark_outcome_signal from track_outcome we need to add it somewhere else in the sentry processing pipe)